batfish.config.js

```diff
@@ -0,0 +1,40 @@
+'use strict';
+
+const path = require('path');
+const navigationStructure = require('./_tmp_assembly/navigation.json');
+
+module.exports = () => {
+  return {
+    dataSelectors: {
+      examplesSubNavigationList: () => {
+        return navigationStructure
+          .navigationList
+          .find((routeConfig) => routeConfig.name === 'Examples')
+          .items;
+      },
+      documentationData: () => {
+        return navigationStructure
+          .navigationList
+          .find((routeConfig) => routeConfig.name === 'Documentation' )
+          .props
+          .documentationData;
+      }
+    },
+    fileLoaderExtensions: (extensions) => extensions.concat(['svg', 'ico']),
+    inlineJs: [{ filename: path.join(__dirname, './dist/assembly.js') }],
+    notFoundPath: path.join(__dirname, './site/pages/404.js'),
+    pagesDirectory: path.join(__dirname, './site/pages'),
+    siteBasePath: '/assembly/',
+    webpackConfigStaticTransform: (webpackConfig) => {
+      // We need to ignore the following two files becasue their contents are executed immediately,
+      // and they rely on DOM APIs, so the static site renderer chokes on them otherwise.
+      webpackConfig.module.rules.unshift({
+        test: /copy\.js$|expander\.js$/,
+        loader: 'ignore-loader'
+      });
```

Comment:
You're too fast, and already submitted a PR for the fix: https://github.com/mapbox/batfish/pull/119

I'd still be happy to create the issue if you want it to exist!